README.md

```diff
@@ -2193,6 +2188,25 @@ _Unofficial libraries for package and dependency management._
 
 **[⬆ back to top](#contents)**
 
+## Reflection
+
+- [autowire](https://github.com/tiendc/autowire) - Dependency injection using Generics and reflection.
```

Comment:
`autowire` , `boot-go`, `dig` and `linker` are better kept in **Dependency Injection** category. as that is what they directly offer regardless of how they work under the hood.